bin/v-import-directadmin

```diff
@@ -0,0 +1,311 @@
+#!/bin/bash
+# info: Import DirectAdmin backup to a new user
+#
+# example: v-import-directadmin /backup/backup.tar.gz
+#
+# Based on sk-da-importer
+# Credits: Maks Usmanov (skamasle) and contributors:
+# Thanks to <https://github.com/Skamasle/sk_da_importer/graphs/contributors>
+
+# This script is provided whitout any warranty
+# Run at your own risk
+# Version 0.2
```

Comment:
```suggestion
# Version 0.1
```

Reply:
Fixed by amend commit, thanks for the feedback!